tests/test_objective_funs.py

```diff
@@ -816,6 +819,64 @@ def test(coil, grid=None):
         test(mixed_coils, grid=[LinearGrid(N=5)] * len(mixed_coils.coils))
         test(nested_coils, grid=nested_grids)
 
+    @pytest.mark.unit
+    def test_quadratic_flux(self):
+        """Test calculation of quadratic flux on the boundary."""
+        t_field = ToroidalMagneticField(1, 1)
+
+        # test that torus (axisymmetric) Bnorm is exactly 0
+        eq = load("./tests/inputs/vacuum_circular_tokamak.h5")
+        obj = QuadraticFlux(eq, t_field)
+        obj.build(eq, verbose=2)
+        f = obj.compute(field_params=t_field.params_dict)
+        np.testing.assert_allclose(f, 0, rtol=1e-14, atol=1e-14)
+
+        # test non-axisymmetric surface
+        eq = desc.examples.get("precise_QA", "all")[0]
```

Comment:
Why use the initial equilibrium from this solve and not the "real" precise QA equilibrium?